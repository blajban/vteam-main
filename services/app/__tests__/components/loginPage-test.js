import React from 'react';
import renderer from 'react-test-renderer';
import LoginPage from '../../components/loginPage';

jest.mock('expo-constants', () => ({
  manifest: {
    hostUri: 'localhost:3000',
  },
}));

test('LoginPage Renders correctly', () => {
  const loginPage = renderer.create(<LoginPage />).toJSON();
  expect(loginPage).toMatchSnapshot();
});
