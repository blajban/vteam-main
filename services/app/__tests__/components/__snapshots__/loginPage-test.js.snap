// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage Renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#2A9D8F",
      "borderRadius": 10,
      "borderWidth": 5,
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    title="Login"
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 25,
          "fontWeight": "bold",
        }
      }
    >
      Logga in
    </Text>
  </View>
</View>
`;
